infra/gcp/terraform/k8s-infra-gcp-gcve/vsphere/scripts/boskos-userdata.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
```

Comment:
I assume we'll document how this is used later? (If I see correctly it's not documented yet)

Reply:
Yes. It is not used yet, just helps to get the next step started.